### ISSUE TITLE
[css2?] Rendering of the <br> element when margins are applied

### ISSUE THREAD

**Issue by emilio** [w3c/csswg-drafts#4856]
TLDR: Somebody reported an interesting issue in their site, which happened to do with differences across browsers in how `<br>` applies (or doesn't) margin: https://bugzilla.mozilla.org/show_bug.cgi?id=1621387

I don't think the rendering of `<br>` is very well specified other than "it introduces a forced line-break" and such... should it honor margin?

Gecko does in: `data:text/html,<p>Foo<br style="margin: 200px"></p>`

It seems to be long-standing behavior, and something that caught my eye is that MDN says this is something that you can do:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br:

> You can set a margin on `<br>` elements themselves to increase the spacing between the lines of text in the block, but this is a bad practice — you should use the line-height property that was designed for that purpose.

Which is odd because not even IE11 seems to do this. Assuming I'm not missing anything, we should probably change Firefox and specify the behavior of other browsers somewhere (where should rendering of the `<br>` element live? html? Ideally it should be less magical...).

Is there any case where `<br>` respects margins in other browsers?

**Comment by fantasai** [comment]
Related material: https://github.com/w3c/csswg-drafts/issues/610

**Comment by gsnedders** [comment]
Also related: https://github.com/whatwg/html/issues/2291

**Comment by frivoal** [comment]
As discussed in the links above, the same is true about `<wbr>`: Firefox lets it have margins, other browsers don't. There's kind of a use case for allowing that, but that use case is better served by https://drafts.csswg.org/css-text-4/#word-boundaries.
